src/immer/withImmer.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/named */
+import { produce, Draft } from 'immer'
+import { atom, WritableAtom } from 'jotai'
+
+export const withImmer = <Value, Update>(anAtom: WritableAtom<Value, Update>) =>
```

Comment:
```suggestion
export const withImmer = <Value>(anAtom: WritableAtom<Value, Value>) =>
```

Sorry to bother you. My suggestion was too naive. Please revert back to this.

Reply:
No, I'm learning and i love suggestions, I'm gonna revert.